release_build_files/readme.md

```diff
@@ -567,10 +567,16 @@ code.
 
 ## Release Notes
 
+### Future Release
```

Comment:
Please change this to the actual version number of the upcoming release at the next opportunity, per http://go/firebase-cpp-release.